libs/ui/inputs/src/lib/ui-inputs.module.ts

```diff
@@ -46,6 +48,7 @@ import { ImageInputComponent } from './image-input/image-input.component'
     CopyTextButtonComponent,
     CheckboxComponent,
     SearchInputComponent,
+    SwitchToggleComponent,
```

Comment:
All the changes in this file won't be needed anymore when switching to standalone component.